integration-tests/src/test/resources/features/PostcodeLookupRequest.feature

```diff
@@ -31,3 +31,15 @@ Feature: valid postcode test
     Examples:
       | testUserDataSheetRowNumber | testPostCode |
       | 197                        | SW1A 2AA     |
+
+  @postcode-lookup
+  Scenario Outline: Rate limit Is returned by OS API?
+    Given user has the test-identity <testUserDataSheetRowNumber> in the form of a signed JWT string
+    When user sends a POST request to session end point
+    And the user performs a postcode lookup for post code "<testPostCode>"
+    And the response HTTP status code is 404
```

Comment:
This says `And` but the step you've defined is `@When("the response HTTP status code is 404")` - should they be aligned? 

I would actually say they should both be `Then`